### PR TITLE
fixed shaped alignment, added model weight extraction

### PR DESCRIPTION
**Pull request by jereliu**
The main thing causing problem was tf.case (maybe you were trying to use tf.cast?). 

Also added method to extract model weight in line 101-104

### CODE REVIEW

File: gLasso/gLasso.py [hunk 1]
```diff
@@ -42,14 +43,14 @@ def gLasso_model(features, labels, mode, params):
         return tf.estimator.EstimatorSpec(mode, predictions=predictions)
 
     # Compute loss.
-    loss = tf.losses.mean_squared_error(labels=tf.case(tf.expand_dims(labels, axis=1), tf.float32),
```
Comment (jereliu) [inline gLasso/gLasso.py:45]: I think maybe you were trying to use `tf.cast`?

Anyways, you probably don't need to explicit type conversion and reshaping here. 

It's better to convert the shape of `response` instead. and cast `label` to float32 when generating the data (see my version of simu_data.py line 67)

File: gLasso/gLasso.py [hunk 2]
```diff
@@ -88,18 +90,22 @@ def main(argv):
...
+            train_x, train_y, args.batch_size),
         steps=args.train_steps)
 
     # Evaluate the model.
     eval_result = classifier.evaluate(
-        input_fn=lambda:simu_data.eval_input_fn(test_x, test_y, args.batch_size))
+        input_fn=lambda: simu_data.eval_input_fn(test_x, test_y, args.batch_size))
 
-    print('\nTest set accuracy: {accuracy:0.3f}\n'.format(**eval_result))
+    # extract variables from model
```
Comment (jereliu) [inline gLasso/gLasso.py:101]: model extraction method added
